cocos2d/actions/tween.js

```diff
@@ -176,8 +176,30 @@ function Tween (target) {
     this._finalAction = null;
     this._target = target;
     this._tag = cc.Action.TAG_INVALID;
+
+    // for time scale
+    this._timeScale = 1
 }
 
+/**
+ * !#en set/get time scale for tween
+ * !#zh 设置/读取 tween 的 time scale (时间缩放参数)
+ * @property timeScale
+ * @type {Number}
+ * @default 1
+ */
+cc.js.getset(Tween.prototype, 'timeScale',
```

Comment:
Please remove timeScale as it's an unrelated feature to pause & resume,  let's keep the PR simple and straight forward